sos/report/plugins/sssd.py

```diff
@@ -23,12 +24,16 @@ class Sssd(Plugin):
     def setup(self):
         self.add_copy_spec([
             "/etc/sssd/sssd.conf",
-            "/var/log/sssd/*",
             "/var/lib/sss/pubconf/krb5.include.d/*",
             # SSSD 1.14
             "/etc/sssd/conf.d/*.conf"
         ])
 
+        # add individual log files
+        logdir = "/var/log/sssd/"
+        for logfile in listdir(logdir):
+            self.add_copy_spec([ logdir + logfile ])
```

Comment:
Probably skip rotated "*-<date>" logs?

Reply:
current solution pack them (if they fit in the limit or --all-logs is used). I thought of it but I choose to pack them and not change the behavior.